test/ReactWrapper-spec.js

```diff
@@ -465,6 +465,30 @@ describeWithDOM('mount', () => {
         ));
         expect(foundNotSpan).to.have.length(0);
       });
+
+      it('should return props object when props() is called', () => {
+        const SFC = function SFC({ data }) {
+          return (
+            <div data-foo={data}>Test SFC</div>
+          );
+        };
+
+        const content = 'blah';
+        const wrapper = mount(<SFC data={content} />);
+        expect(wrapper.props()).to.deep.equal({ 'data' : content });
```

Comment:
tiny nit: `{ data: content }` - ie, remove the quotes and the space before the colon